src/renderer/utils/InstallManager.tsx

```diff
@@ -256,7 +258,64 @@ export class InstallManager {
             }
         }
 
-        const destDir = Directories.inCommunity(addon.targetDirectory);
+        // Find incompatible add-ons
+        // This iterates through the first level of folders of the  MSFS Community folder looking for the manifest.json
+        // file. It compares the manifest.json file content with the configured incompatible add-ons (data.ts) and if it
+        // finds one or more matches, it will issue a warning.
+        console.log('Searching incompatible add-ons');
+        const manifestFileName = 'manifest.json';
+        const comDir = Directories.communityLocation();
+        const incompatibleAddons: AddonIncompatibleAddon[] = [];
+        try {
+            const addonFolders = fs.readdirSync(comDir);
+            for (const entry of addonFolders) {
+                const filePath = path.join(comDir, entry);
+                const stat = fs.statSync(filePath);
+                if (stat.isDirectory()) {
+                    const dirEntries = fs.readdirSync(filePath);
+                    if (dirEntries.includes(manifestFileName)) {
+                        const manifest = JSON.parse(fs.readFileSync(path.join(filePath, manifestFileName), 'utf8'));
+                        for (const item of addon.incompatibleAddons) {
+                            if ((!item.title || manifest.title === item.title) &&
+                                (!item.creator || manifest.creator === item.creator) &&
+                                (!item.package_version || manifest.package_version === item.package_version)
+                            ) {
+                                console.log("!!! %s: %s", manifest.title, item.description);
```

Comment:
**issue:** remove debug output

Reply:
D idn 't consider this as debug output but as a valid log entry when inc. add-ons have been found. Format can be changed of course. '